galgebra/mv.py

```diff
@@ -1048,7 +1048,7 @@ def proj(self, bases_lst):
         return Mv(obj, ga=self.Ga)
 
     def dual(self):
-        mode = ga.Ga.dual_mode_value
+        mode = self.Ga.dual_mode_value
```

Comment:
This was a reference to ga. It won't work after the removal of circular dependency. And given the context, it actually meant self.Ga, just like in other methods. It's a hidden typo which isn't easy to be found before the removal of circular dependency on ga.